Documentation/kubernetes/install.rst

```diff
@@ -50,8 +50,8 @@ filesystem to be automatically mounted when the node boots.
 If you are using systemd to manage the kubelet, another option is to add a
 mountd systemd service on all hosts:
 
-Due to how systemd mounts <link rel="this SO article for example"
-href="https://unix.stackexchange.com/questions/283442/systemd-mount-fails-where-setting-doesnt-match-unit-name">
```

Comment:
I realise this was there before but it might be clearer to be more explicit what "string' is here:
`Due to how systemd mounts filesystems, the moun tpoint path must be reflected in the unit filename.`
(the link can stay where it is, I just can't format it correctly here)